src/common/routes.js

```diff
@@ -95,16 +96,27 @@ export const conferenceEventsHandler = async (req, res) => {
     ParticipantLabel,
     StatusCallbackEvent,
   } = req.body;
+  const {
+    componentUrl,
+    statusCallbackEvents = [],
+  } = options;
```

Comment:
Are either of these options (or both of them) mandatory? If they are, it's better to have them be normal parameters instead of nested in an optional object. From what I'm seeing, componentUrl is mandatory, so maybe the function signature should be `(req, res, componentUrl, options)`?